### DISCUSSION TITLE
Using Aura theme in Figma?

### DISCUSSION THREAD

**Discussion by lukereynoldsSW** [primefaces/community#1505] [Figma UI Kit] [answered]
Howdy, we've just downloaded the PrimeOne Figma UI kit and would love to know if we can use it with the Aura theme? I'm assuming we can just import via token studio and have it update that way? 

If not, is there any way we can ensure design teams and development teams have the same resources/components to deploy?

Cheers.

**Comment by onursenture** [comment, marked as answer]
Hello, for now, the Figma UI Kit has only the Lara theme. We're working on adding Aura in the next releases.

**Reply by lukereynoldsSW** [reply]
Thank you for the update. I don't suppose there is a rough ETA for release?

**Reply by fkunkler-bee** [reply]
@lukereynoldsSW Now there is: https://discord.com/channels/787967399105134612/1268582959519371409/1268659688879423599
